app/config/routing.yml

```diff
@@ -9,9 +9,9 @@ app:
     type:     annotation
     prefix:   /{_locale}
     requirements:
-        _locale: %app_locales%
+        _locale: '%app_locales%'
```

Comment:
Well, it should work the same whether or not you use quotes. So I suggest to report this to the Symfony plugin if it isn't already a known issue.